core/src/commonMain/kotlin/com/ustadmobile/core/controller/SessionDetailListPresenter.kt

```diff
@@ -0,0 +1,66 @@
+package com.ustadmobile.core.controller
+
+import com.ustadmobile.core.util.SortOrderOption
+import com.ustadmobile.core.view.SessionDetailListView
+import com.ustadmobile.core.view.SessionsListView.Companion.ARG_CONTEXT_REGISTRATION
+import com.ustadmobile.core.view.UstadView
+import com.ustadmobile.door.DoorLifecycleOwner
+import com.ustadmobile.door.doorMainDispatcher
+import com.ustadmobile.lib.db.entities.StatementWithSessionDetailDisplay
+import com.ustadmobile.lib.db.entities.UmAccount
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import org.kodein.di.DI
+
+class SessionDetailListPresenter(context: Any, arguments: Map<String, String>,
```

Comment:
Let's call this StatementListPresenter. It is really a list presenter, not a detail presenter.